plugins/junit/src/main/java/org/apache/struts2/StrutsJUnit4TestCase.java

```diff
@@ -170,6 +168,20 @@ protected ActionProxy getActionProxy(String uri) {
         return proxy;
     }
 
+    private void initActionContext(ActionContext actionContext) {

```

Comment:
Could you make this method `protected`? This will allow override it when needed

Reply:
:+1: thanks!